vm/src/obj/objlist.rs

```diff
@@ -895,11 +899,10 @@ impl PyValue for PyListReverseIterator {
 impl PyListReverseIterator {
     #[pymethod(name = "__next__")]
     fn next(&self, vm: &VirtualMachine) -> PyResult {
-        if self.position.get() > 0 {
-            let position: usize = self.position.get() - 1;
-            let ret = self.list.elements.borrow()[position].clone();
-            self.position.set(position);
-            Ok(ret)
+        let list = self.list.elements.borrow();
+        let pos = self.position.fetch_sub(1) as usize - 1;
+        if let Some(ret) = list.get(pos as usize - 1) {
```

Comment:
Is `pos` not `usize` at this stage? Are you sure it should not be `list.get(pos)`?

Reply:
Oh, whoops, I meant to just do that once.